tests/qunit/mock/ajax/get_widgets_config/index.js

```diff
@@ -0,0 +1,12 @@
+import config from 'elementor/tests/qunit/editor/config';
+
+// eslint-disable-next-line no-unused-vars
+export const getWidgetsConfig = ( action, fullParams ) => {
```

Comment:
Maybe remove the param and remove lint comment?

Reply:
Handled.